src/adfh/ADFH.c

```diff
@@ -79,6 +79,16 @@ static hsize_t h5pset_buffer_size_size            = ADFH_CONFIG_DEFAULT;
 static hsize_t h5pset_sieve_buf_size_size         = ADFH_CONFIG_DEFAULT;
 static unsigned h5pset_elink_file_cache_size_size = ADFH_CONFIG_DEFAULT;
 
+/* HDF5 Chunked Dataset Parameters */
+int chunk_ndim = 0;
+hsize_t chunk_dim[ADF_MAX_DIMENSIONS];
+
+/* HDF5 Filter Parameters */
+H5Z_filter_t filter_id;
+size_t filter_nparams;
```

Comment:
Should `filter_nparams` be initialized to 0, `filter_params` to `NULL` and `filter_flags` to 0?